### ISSUE TITLE
Decrease Connect latency

### ISSUE THREAD

**Issue by dennyabrain** [tattle-made/viral-spiral-backend#68]

**Comment by Bhargav-Dave** [comment]
Hey @dennyabrain, should we close this issue now?

**Comment by dennyabrain** [comment]
No. Connect latency is still unresolved. To recreate, launch n clients and have them connect to thr server simulataneously. The connection time increases linearly as n grows. I want to understand how this can be mitigated, it it can be.
